src/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/table/table.js

```diff
@@ -0,0 +1,161 @@
+  };
+
+  render() {
+    const {
+      pageIndex,
+      pageSize,
+      items,
+      totalItemCount,
+      isSearching,
+      filterOptions,
+      selectionConfig: selection,
+      onDelete,
+      onExport,
+      selectedSavedObjects,
+      onQueryChange,
+      onTableChange,
+      goInApp,
+      getEditUrl,
+      onShowRelationships,
+    } = this.props;
+
+    const pagination = {
+      pageIndex: pageIndex,
+      pageSize: pageSize,
+      totalItemCount: totalItemCount,
+      pageSizeOptions: [5, 10, 20, 50],
+    };
+
+    const filters = [
+      {
+        type: 'field_value_selection',
+        field: 'type',
+        name: 'Type',
+        multiSelect: 'or',
+        options: filterOptions,
+      },
+      // Add this back in once we have tag support
+      // {
+      //   type: 'field_value_selection',
+      //   field: 'tag',
+      //   name: 'Tags',
+      //   multiSelect: 'or',
+      //   options: [],
+      // },
+    ];
+
+    const columns = [
+      {
+        field: 'type',
+        name: 'Type',
+        width: '35px',
+        description: `Type of the saved object`,
+        sortable: false,
+        render: (type) => {
+          return (
+            <EuiIcon type={getSavedObjectIcon(type)} size="s"/>
+          );
+        },
+      },
+      {
+        field: 'title',
+        name: 'Title',
+        description: `Title of the saved object`,
+        dataType: 'string',
+        sortable: false,
+        render: (title, object) => (
+          <EuiLink href={getEditUrl(object.id, object.type)}>
+            {title}
+          </EuiLink>
+        ),
+      },
+      {
+        name: 'Actions',
+        actions: [
+          {
+            name: 'In app',
+            description: 'View the relationships this saved object has to other saved objects',
+            icon: 'eye',
+            onClick: object => goInApp(object.id, object.type)
+          },
+          {
+            name: 'Relationships',
+            description: 'View the relationships this saved object has to other saved objects',
+            icon: 'kqlSelector',
+            onClick: object => onShowRelationships(object.id, object.type, object.title)
+          }
+        ]
+      }
+    ];
+
+    return (
+      <Fragment>
+        <EuiSearchBar
+          filters={filters}
+          onChange={onQueryChange}
+          toolsRight={[
+            <EuiButton
+              key="deleteSO"
+              iconType="trash"
+              color="danger"
+              size="s"
+              onClick={onDelete}
+              isDisabled={selectedSavedObjects.length === 0}
+            >
+              Delete
+            </EuiButton>,
+            <EuiButton
+              key="exportSO"
+              iconType="exportAction"
+              size="s"
+              onClick={onExport}
+              isDisabled={selectedSavedObjects.length === 0}
+            >
+              Export
+            </EuiButton>,
+          ]}
+        />
```

Comment:
Consider `EuiSpacer` between the search bar and table, current spacing is really tight.

Reply:
Fixed!